Felienne/hedy
# [NEW FEATURE] Re-directs username on explore page to public profile

**Description**
In this PR we implement a clickable username on the explore page to re-direct the user to the corresponding public profile. If the username doesn't exist a 404 with "doesn't exist or doesn't have a public profile" is returned. This might not be very user-friendly. But looking up (and matching) all usernames with their public profile to prevent the non-existing one from being clickable is really expensive.

**Fix for**
This PR fixes #1706.

**How to test**
Navigate to the explore page and notice the new "view program" button as well as the blue colored username. Click both and verify that they correctly redirect to either the public profile or the shared program page.

![Schermafbeelding 2022-01-25 om 12 41 06](https://user-images.githubusercontent.com/38155424/150970625-30c0e56a-8724-4f58-a96c-82e5b3f245ba.png)
